picard/ui/itemviews.py

```diff
@@ -695,8 +695,8 @@ def mimeTypes(self):
         return ["text/uri-list", "application/picard.album-list"]
 
     def dragEnterEvent(self, event):
+        event.setDropAction(QtCore.Qt.DropAction.CopyAction)
```

Comment:
I wonder why in the reported case `event.mimeData().hasUrls()` would be false. After all it is files being dropped. 

It all looks like the reported issue was supposed to be handled, but for some reason it does not work. Really needs deeper debugging.